src/Elastic.Clients.Elasticsearch/Core/Infer/Field/FieldConverter.cs

```diff
@@ -48,19 +51,19 @@ private static Field ReadObjectField(ref Utf8JsonReader reader)
 		{
 			if (reader.TokenType == JsonTokenType.PropertyName)
 			{
-				var propertyName = reader.GetString();
-				reader.Read();
-
-				switch (propertyName)
+				if (reader.ValueTextEquals(FieldProperty.EncodedUtf8Bytes))
+				{
+					reader.Read();
+					field = reader.GetString();
+				}
+				else if (reader.ValueTextEquals(FormatProperty.EncodedUtf8Bytes))
```

Comment:
For your info, I proposed an ability to pass an encoded text directly to the reader, but it was declined. See dotnet/runtime#86591.